test/e2e/specs/wp-gutenberg-experimental-features-spec.js

```diff
@@ -104,6 +104,25 @@ describe( `[${ host }] Experimental features we depend on are available (${ scre
 				);
 			}
 		);
+
+		// Regression test for https://github.com/Automattic/wp-calypso/pull/48940.
```

Comment:
I guess that if we can say confidently that the number of default block patterns don't change often (does anyone know?), we can also change the check to fail if it sees the number is equal to the number of default block patterns thinking , which would also mean the ETK patterns haven't loaded.